pkg/occlient/occlient.go

```diff
@@ -17,6 +17,28 @@ import (
 
 const ocRequestTimeout = 1 * time.Second
 
+// ocpath stores the path to oc binary
+var ocpath string
+
+var isInitialized bool
+
+func initialize() {
```

Comment:
do we need `isInitialized`? Wouldn't be enough just to test `ocpath`?

we can do all this in this `initialize` function.
oc path 
```go
func initialize() error {
// check if this was already initialized
if ocpath == "" {
 return nil
}

// do initialization
// return err if something went wrong
....

```

Reply:
makes sense, much neater this way, fixed